openreview/conference/templates/recruitResponseWebfield.js

```diff
@@ -34,8 +35,30 @@ function render() {
           '</div>',
         '</div>'
       ].join('\n'));
+    } else if (declined) {
+      // Get invitation to request max load
+      var reduced_load_invitation_id = CONFERENCE_ID + '/-/' + REDUCED_LOAD_INVITATION_NAME;
+      Webfield.get('/invitations', { regex: reduced_load_invitation_id })
+      .then(function(result) {
+        if (result.hasOwnProperty('invitations') && result.invitations.length) {
+          invitation = result.invitations[0];
+          $response.append([
+            '<div class="panel">',
+              '<div class="row">',
+                '<h3>If you chose to decline because the number of reviews required was too high, you can request a reduced reviewer load by clicking here: <a href="/invitation?id=' + reduced_load_invitation_id + '&response=Yes&user=' + args.user + '&key=' + args.key + '">Request reduced load</a></h3>',
```

Comment:
Can you add `style="line-height:normal;"` to the`h3`s. They look a little cramped. Also add a `style="font-weight:bold"` to the `a` tag to make that stand out a little more.

Reply:
Thanks @zbialecki , it looks much better now.